webapp/js/ctrl/main.js

```
@@ -548,75 +545,31 @@ plik.controller('MainCtrl', ['$scope', '$api', '$config', '$route', '$location',
             }
 
             // Get TTL in seconds
-            var ttl = $scope.getTTL();
+            var ttl = getTTL($scope.ttlValue, $scope.ttlUnit);
 
             // Invalid negative value
-            if ($scope.ttlUnit !== 'never' && ttl < 0) ok = false;
+            if ($scope.ttlUnit !== 'unlimited' && ttl < 0) ok = false;
 
             // Check against server side allowed maximum
             maxTTL = $scope.config.maxTTL;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)